tests/test_portchannel_global.rb

```diff
@@ -37,6 +38,11 @@ def cleanup
     config_no_warn "no port-channel load-balance #{ethernet}"
   end
 
+  def n3k_is_t2?
+    return unless /N3/ =~ node.product_id
+    Platform.chassis['descr'].include?('Nexus 31') ? true : false
```

Comment:
Ah, nevermind then.

Looks like 31xx displays a little different for 3132 vs 3172. You can probably match against something like this:

`[/Nexus3000 C31|Nexus 31/]

Validate your tests against n31x-101, n3k-161, and n3k-103.